apps-contrib/resources/kafka-console-consumer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mainClass": "kafka.tools.ConsoleConsumer",
+  "repositories": [
+    "central"
+  ],
+  "dependencies": [
+    "org.apache.kafka::kafka:latest.stable"
+  ],
+  "name": "kafka-console-consumer"
```

Comment:
The `name` field isn't required if it has the same value as the file name (stripping the `.json` extension).

Reply:
Removed the line